app/controllers/miq_ae_class_controller.rb

```diff
@@ -873,10 +873,14 @@ def fields_form_field_changed
           page << javascript_hide("field_default_value")
           page << javascript_show("field_password_value")
           page << "$('#field_password_value').val('');"
+          session[:field_data]["default_value"] =
```

Comment:
@h-kataria I changed keys in session[:field_data] from strings to symbols. Is it right?